networks/local/terradnode/Dockerfile

```diff
@@ -1,11 +1,11 @@
 FROM alpine:3.7
-MAINTAINER Greg Szabo <greg@tendermint.com>
+MAINTAINER YunSuk Yeo <yun@terra.money>
```

Comment:
For localnet test, it will use local docker image which is built from following commands.
$ make build-linux
$ make build-docker-terradnode
$ make localnet-start

MAINTAINER just means writer of Dockerfile (can be skipped)